frontend/src/scenes/paths/PathNodeCardButton.tsx

```diff
@@ -46,6 +46,8 @@ export function PathNodeCardButton({
     }
     const openModal = (): void => openPersonsModal({ path_end_key: name })
 
+    const isTruncatedPath = name.endsWith('...')
```

Comment:
Feels like there's a chance of false positives?

Reply:
yeah, given enough users, someone's going to run into every edge case 😅

Reply:
I made it more specific.